lib/qwerty/source.rb

```diff
@@ -0,0 +1,12 @@
+module Qwerty
+  class Source < Ruote::Participant
+    def on_workitem
+      workitem.fields['source'] = {
+        :font_size => '12px',
+        :font_family => 'Helvetica',
+        :font_weight => '400' 
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.